simulation/halsim_print/publish.gradle

```diff
@@ -0,0 +1,43 @@
+apply plugin: 'maven-publish'
+apply plugin: 'edu.wpi.first.wpilib.versioning.WPILibVersioningPlugin'
+
+if (!hasProperty('releaseType')) {
+    WPILibVersion {
+        releaseType = 'dev'
+    }
+}
+
+def pubVersion = ''
+if (project.hasProperty("publishVersion")) {
+    pubVersion = project.publishVersion
+} else {
+    pubVersion = WPILibVersion.version
+}
+
+def baseArtifactId = 'halsim_print'
+def artifactGroupId = 'edu.wpi.first.hal'
```

Comment:
I think I would prefer the GroupID to be `edu.wpi.first.halsim`. Also, we have usually been using `-` instead of `_` for the baseId separator. But other then that, everything is good.